packages/react-form-state/src/utilities.ts

```diff
@@ -50,3 +50,13 @@ export function set<InputType extends Object>(
     } as InputType;
   }
 }
+
+export function flatMap<T>(
```

Comment:
i suspect you're probably safe to pull in the `esnext.array` lib, unless your goal is to specifically avoid it

Reply:
I was running into some issues with getting the testing environment to recognize `flatMap`. I'll give it another shot.

Reply:
let me know if you want me to have a peek

Reply:
Took a closer look at this, I think the issue lies with our `ts-jest` dependencies/version. Haven't found a conclusive fix, but open to other ideas in the meantime.

Meanwhile I found [this](https://github.com/Shopify/quilt/pull/475), which gives me a little more confidence in handrolling our own `flatMap`, seeing that we wrote our own implementations of other utilities. But having the real thing is probably ideal 👍

Reply:
Ohhhh we need a polyfill. I think we use `core-js` in `graphql-tools-web`.